qp_qiime2/tests/test_qiime2.py

```diff
@@ -419,16 +435,16 @@ def test_filter_samples(self):
         self._clean_up_files.append(out_dir)
 
         params = {
-            'i-table': '8',
-            'p-min-frequency': '5',
-            'p-max-frequency': '10',
-            'p-min-features': '5',
-            'p-max-features': '9223372036854775807',
-            'p-where': ''
+            'BIOM table': '8',
+            'Minimum feature frequency across samples': '5',
+            'Maximum feature frequency across samples': '10',
+            'Minimum features per sample': '5',
+            'Maximum features per sample': '9223372036854775807',
```

Comment:
Same here, these were `int` above but now are `str`?

Reply:
Good catch - the code forces a cast anyways so it doesn't really matter.